### PR TITLE
You can "talk" with exodii workers now

### PR DESCRIPTION
**Pull request by GuardianDll**
#### Summary
None
#### Purpose of change
Something i ought to add long time ago, but i'm a not very dialogue guy
#### Describe the solution
You can talk a bit with workers, and find they ignore you

### CODE REVIEW

**Comment by fairyarmadillo** [comment]
The dialog refers to these as cyborgs, but I'm not sure if the player is meant to be able to identify the workers as such without dissecting them. I _think_ the intended idea is that characters mistake them for robots.

**Comment by I-am-Erk** [comment]
That's a soft one but Rubik's dialogue does imply that you can't tell the exodii are partly organic most of the time.

**Comment by GuardianDll** [comment]
so replace `cyborg` with something else like `robot`?

**Comment by I-am-Erk** [comment]
I like being vague so that it's accurate even if you know theyr cyborgs... Hang on

File: data/json/npcs/exodii/exodii_workers.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+[
+  {
+    "type": "talk_topic",
+    "id": "TALK_EXODII_WORKER",
+    "dynamic_line": [
+      "&You see a human-like robot walking nearby.",
+      "&You see a cyborg moving somewhere in a hurry.",
```
Comment (I-am-Erk) [inline data/json/npcs/exodii/exodii_workers.json:7]:
```suggestion
      "&You see a mechanical humanoid moving somewhere in a hurry.",
```
Stuff like this. Other terms might be "humanoid machine" and stuff.

**Pull request merged**